packages/@aws-cdk/aws-codebuild/test/report-group.test.ts

```diff
@@ -142,4 +143,16 @@ describe('Test Reports Groups', () => {
       "UpdateReplacePolicy": "Delete",
     });
   });
+
+  test('can be created with type=CODE_COVERAGE', () => {
```

Comment:
Please add a test here verifying that `type` defaults to `TEST`.

Reply:
Done!